2-mandatory/4-people-i-know.js

```diff
@@ -386,6 +387,11 @@ Next, I want you to find all of my friends who work for "POWERNET" and then stor
 
```

Comment:
Good job on these. Now that you're comfortable with `for` loops, can you re-write these using array methods? That will be very good practice!